collector/connscrape.cpp

```diff
@@ -63,7 +63,7 @@ int main(int argc, char** argv) {
     std::cout << std::endl
               << "Endpoints:" << std::endl;
     for (const auto& ep : endpoints) {
-      std::cout << " " << ep << " " << ep.pid() << std::endl;
+      std::cout << " " << ep << " " << ep.originator()->getPid() << std::endl;
```

Comment:
This is something I intended to change in my own PR. This is not the best place to print the pid. Instead change the << operator for ContainerEndpoint and Process.

Reply:
Please, feel free to do so; I'll handle the conflicts.